### DISCUSSION TITLE
done a pull request

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#22] [Outreachy 2022] [answered]
@BatoolMM i made my pull request on #33
also is the disussion forum OSCSA forum ?, because when i filled the form i wasnt redirected anywhere , i only got an email which didnt have  a link to maybe a slack community or anything

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi, 

All the questions for Outrechay will be through GitHub Discussion or the issue to keep track of everything!

No worries if you hadn't received anything yet when you completed the forum!
